src/client/staking-client.ts

```diff
@@ -143,7 +143,7 @@ export class StakingClient {
     return StakingService.CreateWorkflow(req, initReq);
   }
 
-  // Get a workflow given its project and workflow id.
+  // Get a workflow given workflow id and project inferred from your API key.
```

Comment:
We can remove the second half of this sentence to make things even simpler:

> // Get a workflow from its ID.

Same simplification applies above as well.

Reply:
Thx. Done